### ISSUE TITLE
[Minio Support] Add more configuration options to S3 output

### ISSUE THREAD

**Issue by mkerins** [fluent/fluent-bit#3188]
**Is your feature request related to a problem? Please describe.**
We use MinIO for on-prem object storage testing which is AWS compatible. I tried setting the `endpoint` configuration value for the S3 output to `https://minio.local:9000` but it fails to resolve because the port is part of the URL. I removed the port and it's defaulting to 443. I also tried adding a `port 9000` line to the S3 output configuration but it defaulted to port 443. We also don't use TLS for these internal services so it would be great if verification could be disabled

**Describe the solution you'd like**
More configuration options for the S3 output:
```
verify_tls       false
endpoint         http://minio.local
endpoint_port    9000
```
Thank you very much.

**Comment by dtaniwaki** [comment]
I'm working on a PR.

**Comment by dtaniwaki** [comment]
@mkerins I implemented the MinIO support feature.
https://github.com/fluent/fluent-bit/pull/3215

Please take a look and give me your feedback about the config.

**Comment by mkerins** [comment]
Thank you very much @dtaniwaki ! Had one comment on the docs PR

**Comment by alfrye** [comment]
what is the status of this issue? was this ever implemented ?

**Comment by patrick-stephens** [comment]
Looks like it was merged under #3215

**Comment by shaojielinux** [comment]
I also encountered the same problem. Have you solved it?
